packages/cms-cli/commands/mv.js

```diff
@@ -0,0 +1,88 @@
+const { moveFile } = require('@hubspot/cms-lib/api/fileMapper');
+const {
+  loadConfig,
+  validateConfig,
+  checkAndWarnGitInclusion,
+} = require('@hubspot/cms-lib');
+const { logger } = require('@hubspot/cms-lib/logger');
+const {
+  logApiErrorInstance,
+  ApiErrorContext,
+} = require('@hubspot/cms-lib/errorHandlers');
+
+const {
+  addConfigOptions,
+  addPortalOptions,
+  addUseEnvironmentOptions,
+  setLogLevel,
+  getPortalId,
+} = require('../lib/commonOpts');
+const { logDebugInfo } = require('../lib/debugInfo');
+const { validatePortal } = require('../lib/validation');
+const { trackCommandUsage } = require('../lib/usageTracking');
+const { isPathFolder } = require('../lib/filesystem');
+
+const loadAndValidateOptions = async options => {
+  setLogLevel(options);
+  logDebugInfo(options);
+  const { config: configPath } = options;
+  loadConfig(configPath, options);
+  checkAndWarnGitInclusion();
+
+  if (!(validateConfig() && (await validatePortal(options)))) {
+    process.exit(1);
+  }
+};
+
+const getCorrectedDestPath = (srcPath, destPath) => {
+  if (!isPathFolder(srcPath)) {
+    return destPath;
+  }
+
+  // Makes sure that nested folders are moved independently
+  return `${destPath}/${srcPath.split('/').pop()}`;
+};
+
+exports.command = 'mv <srcPath> <destPath';
+exports.describe = 'Move a remote file or folder in HubSpot';
+
+exports.handler = async options => {
+  loadAndValidateOptions(options);
+
+  const { srcPath, destPath } = options;
+  const portalId = getPortalId(options);
+
+  trackCommandUsage('mv', {}, portalId);
+
+  try {
+    await moveFile(portalId, srcPath, getCorrectedDestPath(srcPath, destPath));
+    logger.log(`Moved "${srcPath}" to "${destPath}" in portal ${portalId}`);
```

Comment:
Not sure if we have an agreed on convention for this, but should we use `logger.success` for these types of messages?

Reply:
That is a good point, I will make the change.